src/openrct2/world/Sprite.h

```diff
@@ -378,4 +373,23 @@ template<typename T = SpriteBase> class EntityList
     }
 };
 
+class EntityTweener
+{
+    std::vector<SpriteBase*> _ents;
+    std::vector<CoordsXYZ> _prePos;
+    std::vector<CoordsXYZ> _postPos;
```

Comment:
```suggestion
    std::vector<SpriteBase*> Entities;
    std::vector<CoordsXYZ> PrePos;
    std::vector<CoordsXYZ> PostPos;
```
According to our coding guidelines, `_bla` is for local static variables, where as member variables are title case.

Reply:
Thats news to me, the underscore is everywhere.

Reply:
Well, I guess the underscores predates us having a style? Dunno, but that's what is documented here:
https://github.com/OpenRCT2/OpenRCT2/wiki/Coding-Style
```c++
/// Structure members are title case.
/**
 * Represents a currently running animation.
 */
struct PeepAnimation
{
    int32_t AnimationIndex;
    int32_t AnimationLength;
};
```

Reply:
I mean it sort of makes sense for public members, none of those class members are public. Most of the code base has the underscore for private ones.

Reply:
The style was underscore for private members or static global fields (although this should be kept to minimum)

Reply:
So should I leave it as is?

Reply:
Well, Ted did say *was*, so I guess it's what I said (?).
In any case I would really like the change to the ents variable name, at least

Reply:
Change to title case, public members are title case.

Reply:
They are not public tho

Reply:
Leave them with underscore prefixes then.

Reply:
Too late, we need to have an agreement on that at some point. Also should update the Wiki page with more samples and cases like this.